test/listener_test.py

```diff
@@ -216,6 +218,42 @@ def test_reader_writer(self):
             self.assertEqual(msg1, msg2)
             self.assertAlmostEqual(msg1.timestamp, msg2.timestamp)
 
+class canutilsLog(unittest.TestCase):
```

Comment:
I think we should move these new tests to their own files, and rename the Test class.